src/test/java/org/broadinstitute/consent/http/service/EmailServiceTest.java

```diff
@@ -115,6 +116,86 @@ void initService() {
         config);
   }
 
+  @Test
+  void sendMessage() throws Exception {
+    String userEmail = "user@duos";
+    User user = new User();
+    user.setEmail(userEmail);
+    String subject = "subject";
+    var model = Map.of("key", "value");
+    String entityReferenceId = "entityReferenceId";
+    Integer userId = 1234;
+    Integer voteId = 4567;
+    when(templateHelper.getTemplate(EmailType.COLLECT.templateName)).thenReturn(mock());
+    var message = new org.broadinstitute.consent.http.mail.message.MailMessage(user, EmailType.COLLECT) {
+      @Override
+      public String createSubject() {
+        return subject;
+      }
+
+      @Override
+      public Object createModel(String serverUrl) {
+        return model;
+      }
+
+      @Override
+      public String getEntityReferenceId() {
+        return entityReferenceId;
+      }
+
+      @Override
+      public Integer getVoteId() {
+        return voteId;
+      }
+    };
+    Template template = mock();
+    when(templateHelper.getTemplate(EmailType.COLLECT.templateName)).thenReturn(template);
+    Response response = new Response();
+    response.setStatusCode(200);
+    response.setBody("body");
+    when(sendGridAPI.sendMessage(any(), any())).thenReturn(response);
+    String emailText = "emailText";
+    doNothing()
+        .when(template)
+        .process(
+            eq(model),
+            argThat(
+                writer -> {
+                  try {
+                    writer.append(emailText);
+                  } catch (IOException e) {
+                    throw new RuntimeException(e);
+                  }
+                  return true;
+                }));
+
+    Instant fixedInstant = Instant.now();
+    try (var mockedStatic = mockStatic(Instant.class)) {
```

Comment:
This is a way to get deterministic results from `now()` so date and time arguments can be verified without using `any()`.